### ISSUE TITLE
Add "Skip" button 

### ISSUE THREAD

**Issue by e1r0nd** [e1r0nd/LearnWords#59]
Add a button "Skip" in "Repeat" mode:
1. skip current word
2. move it to the previous step
3. show a correct answer

**Comment by hhzl** [comment]
Good idea. This is better than forcing the user to give a wrong answer and then the word would go back to step 1.
